libs/movie/src/lib/movie/pipes/type-dictionary.pipe.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule, Pipe, PipeTransform } from '@angular/core';
+import { ContentType } from '@blockframes/utils/static-model';
+
+const typeDictionary = {
+  tv: {
+    runningTime: 'Average Episode Running Time',
+    runningTimeError: 'Please fill in a valid runtime.',
+    runningTimeHint: 'Please specify the average duration of an episode.'
+  },
+  movie: {
+    runningTime: 'Running Time',
+    runningTimeError: 'Please specify your project\'s runtime.',
+    runningTimeHint: 'In minutes.'
+  }
+}
```

Comment:
Let's use better naming. Don't forget to update in the template file: 
```suggestion
const runningTime = {
  tv: {
    label: 'Average Episode Running Time',
    error: 'Please fill in a valid runtime.',
    hint: 'Please specify the average duration of an episode.'
  },
  movie: {
    label: 'Running Time',
    error: 'Please specify your project\'s runtime.',
    hint: 'In minutes.'
  }
};
```